dash_core_components/_checklist.py

```diff
@@ -0,0 +1,64 @@
+"""
+Autogenerated file
+DO NOT EDIT.
+"""
+import typing
+
+from dash_component_system import (
+    DashComponent, UNDEFINED, Undefined, ComponentProp
+)
+
+
+class Checklist(DashComponent):
+    """
+    Checklist is a component that encapsulates several checkboxes. The values
+    and labels of the checklist is specified in the `options` property and the
+    checked items are specified with the `values` property. Each checkbox is
+    rendered as an input with a surrounding label.
+    """
+    _namespace = 'dash_core_components'
+    _typename = 'Checklist'
+    available_wildcard_properties = [
+
+    ]
+    id = ComponentProp('id', UNDEFINED, False)
+    options = ComponentProp('options', "[]", False)
+    values = ComponentProp('values', UNDEFINED, False)
+    className = ComponentProp('className', UNDEFINED, False)
+    style = ComponentProp('style', UNDEFINED, False)
+    inputStyle = ComponentProp('inputStyle', "{}", False)
+    inputClassName = ComponentProp('inputClassName', '', False)
+    labelStyle = ComponentProp('labelStyle', "{}", False)
+    labelClassName = ComponentProp('labelClassName', '', False)
+    loading_state = ComponentProp('loading_state', UNDEFINED, False)
+
+    def __init__(
+        self,
+        id=UNDEFINED,
+        options="[]",
```

Comment:
@T4rk1n is it intentional that these empty objects are quoted (`"[]"`, `"{}"`) here and in `ComponentProp`?

Reply:
No, default props generation needs more testing.